### ISSUE TITLE
Formating a file (ctrl+shift+f) truncate last character of lines

### ISSUE THREAD

**Issue by Kiiv** [grosenberg/fluentmark#13]
When formating files in Eclipse (shortcut ctrl+shift+f), last character of lines are truncate.

To reproduce, insert the following text in a md file and it ctrl+shift+f :
```
### Test

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```

**Comment by grosenberg** [comment]
Works as expected -- no loss of characters anywhere. If this is a reproducible problem, please report version of Java, Eclipse and type of system.

**Comment by Kiiv** [comment]
Hi !
Thanks for your answer.

Ok, I tried to reproduce in an other environment without succes.
So :
- Working environment :
Windows 10
Eclipse Neon 4.6.3
Jdk Orable 8

- Not working environment :
Debian Jessie
Eclipse Neon 4.6.3
Jdk Oracle 8

**Comment by grosenberg** [comment]
Fixed with v0.12.0.
